diff --git a/setup.py b/setup.py
index 190c339..3abd611 100644
--- a/setup.py
+++ b/setup.py
@@ -70,11 +70,11 @@ if len(sys.argv) > 1 and sys.argv[1] == '--fakedependency':
     del sys.argv[1]
     install_requires += ["fakedependency >= 1.0.0"]
 
-__requires__ = install_requires[:]
+#__requires__ = install_requires[:]
 
-egg = os.path.realpath('setuptools-0.6c16dev6.egg')
-sys.path.insert(0, egg)
-import setuptools; setuptools.bootstrap_install_from = egg
+#egg = os.path.realpath('setuptools-0.6c16dev6.egg')
+#sys.path.insert(0, egg)
+#import setuptools; setuptools.bootstrap_install_from = egg
 
 from setuptools import setup
 from setuptools.command import sdist
@@ -454,7 +454,7 @@ setup(name=APPNAME,
                                       "static/css/*.css",
                                       ]
                     },
-      setup_requires=setup_requires,
+      #setup_requires=setup_requires,
       entry_points = { 'console_scripts': [ 'tahoe = allmydata.scripts.runner:run' ] },
       zip_safe=False, # We prefer unzipped for easier access.
       **setup_args
